cli/cli/command_framework/lowlevel/flags/flag_type_processors.go

```diff
@@ -45,6 +46,27 @@ func processStringFlag(
 	return nil
 }
 
+func processStringSliceFlag(
+	flagKey string,
+	shorthand string,
+	defaultValueStr string,
+	usage string,
+	cobraFlagSet *flag.FlagSet,
+) error {
+
+	if defaultValueStr != "" {
```

Comment:
There's an assumption in the library that defaults will always derive from strings.  

We could look to parse string representations of arrays, or add more genral support e.g. for `[]string` here.  

However, my thinking is defaults don't make much sense for a slice flag, so just make it clear it won't work, where somebody does specify a default.